### DISCUSSION TITLE
Donde subir el informe?

### DISCUSSION THREAD

**Discussion by Burbina200** [IIC2133-PUC/2024-1#48] [Tarea 0] [answered]
Hola! Me gustaría confirmar dónde debo subir el informe. ¿Se debe cargar en el mismo repositorio?

**Comment by benjavicente** [comment, marked as answer]
Sí.

<img width="926" alt="image" src="https://github.com/IIC2133-PUC/2024-1/assets/62021328/40b7ccbf-53c0-49c2-b539-0ddbfa612a08">
